### PR TITLE
Fix dockerfile and readme

### PR DESCRIPTION
**Pull request by willbasky**

### CODE REVIEW

File: docker/front/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,10 @@ ENV LC_ALL=C.UTF-8
 # NODE_ENV used to run front server on production
 ENV NODE_ENV=production
 
+# These variables are for independent from front/config.js
```
Comment (neongreen) [inline docker/front/Dockerfile]: Weird English

File: front/README.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
- - `cd dist`
- - `npm i`
- - `node server`
+  Requirements: Node version should be >= 11, because of using fs async methods which experimental now.
+
+1. `git pull`
+2. `npm i`
+
+3. Now you have a choice how to set PORT and API_URL.
+   First, set them in front/config.js once.
```
Comment (neongreen) [inline front/README.md]: > Either set them in `front/config.js`, or use export ...

File: docker/front/Dockerfile (hunk 1 above)
Comment (neongreen) [reply docker/front/Dockerfile]: Do these variables override ones in front/config.js, or do they serve as defaults when front/config.js doesn't have any variables set?

Comment (willbasky) [reply docker/front/Dockerfile]: Both.

Comment (neongreen) [reply docker/front/Dockerfile]: If front/config.js says that port is 4444 and the shell variable says PORT=5555, which value will actually be used?

Comment (willbasky) [reply docker/front/Dockerfile]: 5555

Comment (neongreen) [reply docker/front/Dockerfile]:
```suggestion
# Overrides for variables in front/config.js
```

**Review by neongreen — APPROVED** [review]

**Pull request merged**